internal/pkg/template/env.go

```diff
@@ -11,11 +11,11 @@ import (
 const (
 	// EnvCFTemplatePath is the path where the cloudformation for the environment is written.
 	EnvCFTemplatePath       = "environment/cf.yml"
-	fmtEnvCFSubTemplatePath = "environment/cf/%s.yml"
+	fmtEnvCFSubTemplatePath = "environment/partials/%s.yml"
```

Comment:
Is this different from the services templates directory where we use `common` for the fragments?

Reply:
It's pretty much the same thing, I didn't want to name it `common` since we don't have multiple environment types, but I think it also makes sense to rename `common` to `partials` for services